Scoop/Controller/ProfileViewController.swift

```diff
@@ -17,17 +17,33 @@ class ProfileViewController: UIViewController {
     private let travelingStackView = UIStackView()
     private let favoritesStackView = UIStackView()
     
-    private let user = NetworkManager.shared.currentUser
-
+    private var user = NetworkManager.shared.currentUser
+    
+    private var hometown: String?
+    private var talkative: Float?
+    private var music: Float?
+    private var snack: String?
+    private var song: String?
+    private var stop: String?
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         view.backgroundColor = .systemGray5
         
-        setupHeaderImage()
-        setupContainerView()
-        setupEditButton()
-        setupProfileImageView()
-        setupProfileStackView()
+        NetworkManager.shared.getUser { result in
+            switch result {
+            case.success(let user):
+                self.user = user
+                self.getUserPreferences()
+                self.setupHeaderImage()
+                self.setupContainerView()
+                self.setupEditButton()
+                self.setupProfileImageView()
+                self.setupProfileStackView()
```

Comment:
Should this stuff be done in the main thread?

Reply:
I essentially have to restrucutre the whole View, so I think I'm gonna include it all in the next PR for Profie HF